### PR TITLE
Message View (UI only) - Notifications - Minor fixes

### PR DESCRIPTION
**Pull request by kormic**
Message View (UI only), popover notifications for text inputs, Roboto fonts, Fixed timestamps (you can double tap a message to toggle timestamp), added an extension for UIColor to get colours with hex values and some minor fixes....

### CODE REVIEW

**Comment by gsaslis** [comment]
@kormic: please include a title a description when opening a new PR, so it is easier to understand what we're looking at here... 'merge request' doesn't really tell us much about what you've been building here... ;)

**Comment by kormic** [comment]
@gsaslis Duh!!! I m sorry once again!!! I put that title while I was checking my commits and then I forgot to change it! Forgive my noobiness!!!

**Comment by gsaslis** [comment]
:D dude - keep the title short, use/edit the description for details!  ; )   ; )

**Comment by kormic** [comment]
One day I will learn!

**Comment by gsaslis** [comment]
**every** day we learn! ;)

File: UIColor+hexValue.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//  Created by Kornelakis Michael on 9/2/15.
+//  Copyright © 2015 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+
+
+extension UIColor {
+    
+    static func colorWithHexValue(redValue: CGFloat, greenValue: CGFloat, blueValue: CGFloat, alpha: CGFloat) -> UIColor {
```
Comment (rafaelks) [inline UIColor+hexValue.swift]: What about creating an extension to return Rocket.Chat colors? Something like...

1. `static func rocketMainFontColor() -> UIColor`
2. `static func rocketSecondaryFontColor() -> UIColor`
3. `static func rocketRedColor() -> UIColor`
4. `static func rocketBlueColor() -> UIColor`

Comment (gsaslis) [inline UIColor+hexValue.swift]: @rafaelks :+1:

Comment (kormic) [inline UIColor+hexValue.swift]: OK, do we have the hex values for these colors?

Comment (gsaslis) [inline UIColor+hexValue.swift]: I believe you can get them from the web interface, but I think I'd also used them in my leftMenu.storyboard somewhere... ; )

Comment (rafaelks) [inline UIColor+hexValue.swift]: You're already using some colors into the code... using that colors is a great start! :+1:

**Comment by rafaelks** [comment]
Cannot be merged.

**Comment by kormic** [comment]
@rafaelks Conflicts resolved. Feel free to review...

File: UIColor+hexValue.swift [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    // hex #04436A
+    static func rocketDarkBlueColor() -> UIColor {
+        
+        return UIColor(red: 4/255.0, green: 67/255.0, blue: 106/255.0, alpha: 1)
+
+        
+    }
+    
+    
+}
```
Comment (rafaelks) [inline UIColor+hexValue.swift]: :+1:

**Pull request merged**